python/lib/communication/pyproject.toml

```
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["setuptools >= 70.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "dmod.communication"
+authors = [
+    { name = "Robert Bartel" },
+    { name = "Austin Raney", email = "austin.raney@noaa.gov" },
+    { name = "Chris Tubbs" },
+    { name = "Nels Frazier" },
+    { name = "Chris Tubbs" },
+    { name = "Shengting Cui" },
+]
+dependencies = [
+    "dmod.core>=0.17.0",
+    "websockets>=10.1",
+    "jsonschema",
+    "redis",
+    "pydantic>=1.10.8,~=1.10",
+    "Deprecated",
+    "ngen-config@git+https://github.com/noaa-owp/ngen-cal@master#egg=ngen-config&subdirectory=python/ngen_conf",
+]
+readme = "README.md"
+description = "Communications library package for components of the National Water Model as a Service architecture"
```

Let's take the opportunity to change this also.

```suggestion
description = "Communications library package for components of the DMOD architecture"
```